test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -12898,8 +12898,9 @@ def isUp(self, max_attempts=None):
             try:
                 nodes = self.server.status(NODE, id=self.shortname)
                 if nodes:
-                    self.server.expect(NODE, {'state': 'free'},
-                                       id=self.shortname)
+                    self.server.expect(NODE, {'state': 'free',
+                                              'state': 'provisioning'},
+                                       id=self.shortname, attrop=PTL_OR)
```

Comment:
Thing is that the test itself is calling mom.restart(). It is not calling mom.isUp(). 
It will be semantically odd if I pass in mom state to mom.restart()